beacon-chain/rpc/eth/validator/handlers.go

```diff
@@ -26,11 +29,7 @@ func (s *Server) GetAggregateAttestation(w http.ResponseWriter, r *http.Request)
 	}
 
 	if err := s.AttestationsPool.AggregateUnaggregatedAttestations(r.Context()); err != nil {
-		errJson := &network.DefaultErrorJson{
-			Message: "Could not aggregate unaggregated attestations: " + err.Error(),
-			Code:    http.StatusBadRequest,
-		}
-		network.WriteError(w, errJson)
+		shared.HandleHTTPError(w, "Could not aggregate unaggregated attestations: "+err.Error(), http.StatusBadRequest)
```

Comment:
interesting so instead of having it as network have it as shared.

Reply:
good point, it should go into network